package.json

```diff
@@ -18,146 +18,146 @@
   },
   "dependencies": {
     "@material-ui/core": "^3.9.3",
-    "@material-ui/icons": "^3.0.1",
+    "@material-ui/icons": "^3.0.2",
     "@material-ui/styles": "^3.0.0-alpha.10",
-    "@types/classnames": "^2.2.6",
-    "@types/enzyme": "^3.1.15",
-    "@types/fuzzaldrin": "^2.1.1",
+    "@types/classnames": "^2.2.9",
+    "@types/enzyme": "^3.10.2",
+    "@types/fuzzaldrin": "^2.1.2",
     "@types/lodash-es": "^4.17.3",
-    "@types/moment-timezone": "^0.5.9",
+    "@types/moment-timezone": "^0.5.12",
     "@types/react-dropzone": "^4.2.2",
-    "@types/react-helmet": "^5.0.7",
-    "@types/react-infinite-scroller": "^1.2.0",
+    "@types/react-helmet": "^5.0.8",
+    "@types/react-infinite-scroller": "^1.2.1",
     "@types/react-sortable-tree": "^0.3.6",
-    "@types/storybook__addon-storyshots": "^3.4.3",
-    "@types/string-similarity": "^1.2.0",
+    "@types/storybook__addon-storyshots": "^3.4.9",
+    "@types/string-similarity": "^1.2.1",
     "@types/url-join": "^0.8.3",
-    "apollo": "^2.8.1",
-    "apollo-cache-inmemory": "^1.3.11",
-    "apollo-client": "^2.4.7",
+    "apollo": "^2.15.0",
+    "apollo-cache-inmemory": "^1.6.2",
+    "apollo-client": "^2.6.3",
     "apollo-client-preset": "^1.0.6",
-    "apollo-link": "^1.2.5",
-    "apollo-link-batch-http": "^1.2.5",
-    "apollo-link-context": "^1.0.10",
-    "apollo-link-error": "^1.1.2",
+    "apollo-link": "^1.2.12",
+    "apollo-link-batch-http": "^1.2.12",
+    "apollo-link-context": "^1.0.18",
+    "apollo-link-error": "^1.1.11",
     "apollo-upload-client": "^9.1.0",
     "bootstrap": "4.3.1",
     "classnames": "^2.2.6",
     "crc-32": "^1.2.0",
     "detect-passive-events": "^1.0.4",
-    "downshift": "^1.31.14",
+    "downshift": "^1.31.16",
     "draft-js": "^0.10.5",
-    "draftail": "^1.1.0",
+    "draftail": "^1.2.1",
     "draftjs-to-html": "^0.8.4",
-    "dropzone": "^5.2.0",
+    "dropzone": "^5.5.1",
     "fast-array-diff": "^0.2.0",
     "fuzzaldrin": "^2.1.0",
-    "graphql": "^14.0.2",
-    "graphql-tag": "^2.10.0",
-    "i18next": "^11.2.2",
-    "i18next-browser-languagedetector": "^2.2.0",
+    "graphql": "^14.4.2",
+    "graphql-tag": "^2.10.1",
+    "i18next": "^11.10.2",
+    "i18next-browser-languagedetector": "^2.2.4",
     "i18next-xhr-backend": "^1.5.1",
     "invariant": "^2.2.4",
     "is-url": "^1.2.4",
-    "jquery": "^3.4.0",
+    "jquery": "^3.4.1",
     "jquery.cookie": "^1.4.1",
     "jss": "^9.8.7",
     "keycode": "^2.2.0",
-    "lazysizes": "^4.0.1",
-    "lodash": "^4.17.11",
-    "lodash-es": "^4.17.11",
+    "lazysizes": "^4.1.8",
+    "lodash": "^4.17.14",
+    "lodash-es": "^4.17.14",
     "materialize-css": "^0.100.2",
     "medium-editor": "^5.23.3",
-    "mobx": "^3.0.2",
-    "mobx-react": "^4.1.0",
-    "moment-timezone": "^0.5.21",
-    "popper.js": "^1.14.6",
-    "prop-types": "^15.6.0",
-    "qs": "^6.5.2",
+    "mobx": "^3.6.2",
```

Comment:
Do we need `mobx` and `mobx-react`? We used to have it a long time ago in the old storefront but now I think we don't use it anymore.

Reply:
We do... in variant picker 🤷‍♂️